tests/Api/Responses/Expected/shop/get_channel_by_code.json

```diff
@@ -0,0 +1,14 @@
+{
+    "@context": "\/api\/v2\/contexts\/Channel",
```

Comment:
all responses should be suffixed with `response`, check different files in this catalog